network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -100,10 +100,10 @@ class BlogIndexPage(IndexPage):
     content_panels = IndexPage.content_panels + [
         InlinePanel(
             'featured_pages',
-            label='Featured Posts',
-            help_text='Choose two blog pages to feature',
-            min_num=0,
-            max_num=2,
+            label='Featured',
```

Comment:
Hi @TheoChevalier!

I was wondering if you could please take a look at how we are setting up this "featured_pages" field to be a SynchronizedField [here](https://github.com/mozilla/foundation.mozilla.org/blob/db8245e82d4736663567cf11900da64876993b19/network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py#L118) and let me know if its missing anything?

Asking because I am experiencing a discrepancy in the ordering between the english page and the french page when trying to translate. 

For example, lets say on the english page I had the posts ordered [1,2,3,4,5], and translated the page in french.

If I reorder the posts in the english page to [5,4,3,2,1] and save, the french page still continues to have the original ordering.

Thank you very much in advance!

Reply:
Hey @danielfmiranda, I’ve found that everything works here, you just need to sync translations after publishing the changes in English:

<img width="783" alt="Capture d’écran 2022-07-07 à 09 50 23" src="https://user-images.githubusercontent.com/1294206/177721278-2c188841-2fe2-4dcd-9a24-73523e4639fe.png">
(Menu label is "Sync translated pages")


Then check the "Publish changes now" on the next screen
<img width="737" alt="Capture d’écran 2022-07-07 à 09 50 33" src="https://user-images.githubusercontent.com/1294206/177721539-1927fecb-eaec-4f65-8f4c-5fc87744fa16.png">


Let me know if you need more help testing this!

Reply:
thanks @TheoChevalier for checking!!